complaints/models.py

```diff
@@ -34,7 +34,40 @@ class Complaint(models.Model):
         (CATEGORY_3, "Sophos Firewall"),
     )
 
+    KRISHNA = "K"
+    RAM = "R"
+    SHANKAR = "S"
+    GANDHI = "G"
+    BUDH = "B"
+    VYAS = "V"
+    GAUTAM = "GT"
+    VALMIKI = "VL"
+    MEERA = "ME"
+    MALVIYA = "ML"
+    VISHWAKARMA = "VK"
```

Comment:
Add VK boys and VK Girls as two separate hostels.That will make a total of 3 hostels for girls.

Reply:
I am making it like

    VK_BOYS = "VB"
    VK_GIRLS = "VG"
Is that OK?

Reply:
are u gonna display these VB and VG to the staff members as the bhavan name?

Reply:
Thats upto the frontend

Reply:
If these acronyms are not going to be displayed ,then these are fine.Otherwise change them.
Discuss this with @0xVikas and then decide.